free_court/src/components/Calendar/Calendar.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+
+import './styles.css';
+import { CalendarBox } from '../CalendarBox';
+
+
+
+class CalComponent extends Component {
+
+  createTable = () => {
+      let table = []
+      // Outer loop to create parent
+      for (let i = 5; i < 25; i++) {
```

Comment:
This is dependent on the hours showing 5 to 25. Make this component independent on how many hours are required for showing.